gluonnlp/loss/activation_loss.py

```diff
@@ -0,0 +1,129 @@
+# coding: utf-8
```

Comment:
maybe name this file regularizer.py?

Reply:
yes, renamed to activation_regularizer.py, it is more specific.